maat-court-data-api/build.gradle

```diff
@@ -154,6 +154,9 @@ dependencies {
     testImplementation 'org.testcontainers:junit-jupiter:1.16.0'
     testImplementation 'org.awaitility:awaitility:4.2.0'
 
+    // for retry integration tests on Mac M1 hardware
```

Comment:
As discussed I think we can remove this for now.

Reply:
Done.